kuma/wiki/templates/wiki/document.html

```diff
@@ -390,7 +391,10 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                 </div>
               {% endif %}
             </div>
-
+	    {% if is_admin %}
+	    <center>Document number : {{ document.id }}</center>
```

Comment:
Should the new div be included within the "wiki-block contributors" div or outside it?